internal/winapi/winapi.go

```diff
@@ -0,0 +1,147 @@
+package winapi
```

Comment:
It would be good to have a package doc comment here (e.g. "Package winapi contains various low-level bindings to Windows APIs. It can be thought of as an extension to `golang.org/x/sys/windows`.").